### DISCUSSION TITLE
Steps per revolution and equilibrium/holding rotor positions

### DISCUSSION THREAD

**Discussion by IhorNehrutsa** [CAP1Sup/Intellistep#61] [Q&A]
```
----------------------------------------------------------
|                   | Intellistep steps | BTT steps      |
| MICROSTEP_DIVISOR | per revolution    | per revolution |
----------------------------------------------------------
|          1        |        200        |  256 is absent |
|          2        |        400        |  512           |
|          4        |        800        | 1024           |
|          8        |       1600        | 2048           |
|         16        |       3200        | 4096           |
|         32        |       6400        | 8192           |
----------------------------------------------------------
```
I can't understand how the BTT makes steps.

But I think that 400-800-...-6400 are more wrong way.
In this case motor moves through the rotor equilibrium positions every MICROSTEP_DIVISOR times.
See figure 2.2 and 2.3 http://homepage.cs.uiowa.edu/~jones/step/physics.html
step*S+S/2 is the rotor equilibrium positions in ideal model.
In other side, motor moves through the rotor holding positions every MICROSTEP_DIVISOR times.

In BTT case motor moves through the rotor equilibrium positions much less often. But through the rotor holding positions much less often too.

P.S. I changed my mind right in the process of writing this post in favor of BTT. It seems I am self in equilibrium positions. I doubt now.

**Comment by CAP1Sup** [comment]
What the heck? The motors are rated for 200 full steps/ rotation. How is BTT pulling this out of their butt? The shaft positioning should be much stronger if we use the motors natural full steps
